ultraviolet/shared/src/main/scala/ultraviolet/datatypes/ShaderPrinter.scala

```diff
@@ -302,6 +302,18 @@ object ShaderPrinter:
           case _ =>
             List(s"${render(genType).mkString}.$swizzle")
 
+      case MultiStatements(typeOf, vals) =>
+        val lead = render(typeOf).headOption.map(s => s"$s ").getOrElse("")
```

Comment:
I think rather than empty string, I tend to use `void`.

Reply:
It prints `for (INIT; COND; void NEXT)` like

```diff
+for(int i=1,j=2;i<steps;void i=i+1,j=i){
-for(int i=1,j=2;i<steps;i=i+1,j=i){
```
which appears broken.


Since NEXT is built  from:
```scala
        val n = ShaderAST.MultiStatements(
          ShaderAST.Empty(), // THIS
          (valNames zip nextArgs).map { (varName, next) =>
            ShaderAST.Assign(
              ShaderAST.DataTypes.ident(varName),
              walkTerm(next, envVarName)
            )
          }
        )
```

Reply:
Oh I see, thanks.